src/Context/ComponentOne.js

```diff
@@ -0,0 +1,14 @@
+import { createContext } from "react"
+import { ComponentTwo } from "./ComponentTwo"
+
+export const UserContext = createContext()
```

Comment:
Context should be created in a separate file.

You only showed how to read from context, you also need to add how to set to context.

Reply:
done